sdk/core/azure-core-amqp/src/models/amqp_error.cpp

```diff
@@ -3,6 +3,7 @@
 
 #include "azure/core/amqp/internal/models/amqp_error.hpp"
 
+#include "../amqp/private/unique_handle.hpp"
```

Comment:
IMHO there is no guidance.

IMHO we use relative paths when they are needed. Which in practice means directories that aren't in the "inc" hierarchy.

Reply:
I'd say we better not use them in public headers, because vcpkg may install them to a directory structure that does not look like the one in our repo.
In src, there is not currently a way to avoid (although with some cmake directives, brobably can be done - but I don;t think it is worth it - I look at these usages of `..`, and I don't see how they could break after installation.)